README.md

```diff
@@ -15,7 +15,14 @@ You can use this action to create a namespace in Okteto Cloud as part of your au
 
 **Required**  The name of the Okteto namespace to create.
 
-> Remember that the namespace name must have your github ID as a suffix.
+> Remember that the namespace name must have your github ID as a suffix if you're using Okteto Cloud
+
+### `share`
```

Comment:
Should we call it `members` to be consistent with the CLI? No strong opinion, just though about it. I think if users are used to work with the CLI, it would be easier for them

Reply:
I thought about it, but decided on share because that's the action name in the UI. What do you think @pchico83 @melissa-okteto

Reply:
@rlamana ^^

Reply:
We keep being inconsistent across the UI, the CLI, etc. It might be `members` in the CLI, but in the UI we call it `Share`... Which one do you guys prefer to unify this? I like `share` because it is an actual **action**.

<img width="94" alt="Screenshot 2021-06-16 at 23 01 25" src="https://user-images.githubusercontent.com/237819/122292941-ceb24b80-cef6-11eb-9b53-8bb9901c9e74.png">

Reply:
I like share better as well. We can update the CLI command to match this as well if we want to use the same word in CLI, UI and github action. 

Another option is to create a separate action call "share-namespace", that hides the mechanics.

Reply:
I also like share because it's a clear action word.

Reply:
For me, using a verb/action as a parameter with value is kind of weird. When I usually see a verb as a parameter, it is to represent a flag. `destroyVolumes`, `force`, `build` or `watch` are some examples (but I have to say that probably you are more used to work with CLIs or tools like Github actions than me). I see the Github action as a command (action), so for me, the most natural would be:
* Create this space with these people/members/whatever
* Share this space with these people/members/whatever

I'm not saying that I'm in favor of creating a sharing-namespace action, I just want to say that I think we are mixing actions/commands with parameters/properties. In the UI is called `share`, because it is the main action (or the command to execute).

Anyways, sometimes I like to think too much about things. As I said, I don't have a strong opinion and everybody prefers `share`, so I would leave it as it is :)

Reply:
I agree with @ifbyol. If it is an action, I would go with `share`. If it is a property of the `create namespace` action, I would go with `members`. But I am ok with any option

Reply:
ok, let's go with `members` in that case!